crates/uv-python/src/version_files.rs

```diff
@@ -69,7 +70,13 @@ impl PythonVersionFile {
         options: &DiscoveryOptions<'_>,
     ) -> Result<Option<Self>, std::io::Error> {
         let Some(path) = Self::find_nearest(working_directory, options) else {
-            return Ok(None);
+            // Not found in directory or its ancestors. Looking in user-level config.
+            return Ok(match user_uv_config_dir() {
+                Some(user_dir) => Self::discover_user_config(user_dir, options)
```

Comment:
I wonder if we should co-locate this with the `uv.toml`? Like, if https://github.com/astral-sh/uv/blob/ba74b9ea93b676e7d0b84c386b87e69017a4c27b/crates/uv-dirs/src/lib.rs#L140 exists and a user-level config does not should we prefer that? I'm leaning away from it right now, but wanted to put it up there. Maybe can can consider a `--system` flag separately in the future once we have more feedback.

Reply:
That's an interesting question. But I agree we should consider it separately from this PR.